doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,589 @@
+    Macros :c:macro:`BT_MESH_LIGHT_XYL_SRV_INIT` and :c:macro:`BT_MESH_LIGHT_HSL_SRV_INIT` now take a pointer to the :c:struct:`bt_mesh_lightness_srv` instance instead.
+    Make sure to instantiate the Light Lightness Server if you are using any of these models.
+
+Matter
+------
+
+* Added:
+
+  * :ref:`Thingy:53 weather station <matter_weather_station_app>` application.
+  * :ref:`Template <matter_template_sample>` sample with a guide about :ref:`ug_matter_creating_accessory`.
+  * :ref:`ug_matter_tools` page with information about building options for Matter controllers.
+  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Matter.
+  * :doc:`Matter documentation set <matter:index>` based on the documentation from the Matter submodule.
+
+Thread
+------
+
+* Added:
+
+  * Added support for nRF21540 for nRF52 Series and nRF53 Series, including Bluetooth® LE in the multiprotocol configuration.
+  * Improvements in :ref:`thread_ug_supported_features_v12`:
+
+    * Thread 1.2 supported in all samples.
+    * Retransmissions are now supported when transmission security is handled by the radio driver.
+    * Support for CSL Accuracy TLV in the MLE Parent Response.
+    * Link Metrics data is now properly updated when using ACK-based Probing.
+
+  * Added support for Thread Backbone Border Router on the :ref:`thread_architectures_designs_cp_rcp` architecture.
+
+* Updated:
+
+  * :ref:`ot_cli_sample` sample - Updated with the USB support.
+  * :ref:`ot_coprocessor_sample` sample - Updated with the USB support.
+  * Removed ``NET_SHELL`` from the Thread samples due to its limited usefulness.
+
+Zigbee
+------
+
+* Added:
+
+  * Added production support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF52 Series devices and nRF21540 EK.
+  * Added development support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF53 Series devices and nRF21540 EK.
+  * Added development support for ``nrf5340dk_nrf5340_cpuapp`` to the :ref:`zigbee_ncp_sample` sample.
+  * :ref:`lib_zigbee_zcl_scenes` library with documentation.
+    This library was separated from the Zigbee light bulb sample.
+  * :ref:`zigbee_template_sample` sample.
+    This minimal Zigbee router application can be used as the starting point for developing custom Zigbee devices.
+  * Added API for vendor-specific NCP commands.
+    See the :ref:`Zigbee NCP sample <zigbee_ncp_vendor_specific_commands>` page for more information.
+  * Added API for Zigbee command for getting active nodes.
+
+* Updated:
+
+  * ZBOSS Zigbee stack to version 3.8.0.1+4.0.0.
+    See the :ref:`nrfxlib:zboss_changelog` in the nrfxlib documentation for detailed information.
+  * :ref:`ug_zigbee_tools_ncp_host` to the production-ready v1.0.0.
```

Comment:
Change to: ":ref:`ug_zigbee_tools_ncp_host` is now supported for production and moved to version 1.0.0."